lambdas/retrieve-cri-credential/src/test/java/uk/gov/di/ipv/core/retrievecricredential/RetrieveCriCredentialHandlerTest.java

```diff
@@ -630,6 +630,7 @@ void shouldReturnJourneyEvaluateResponseOnSuccessfulPendingCriResponse() {
                 .thenReturn(
                         VerifiableCredentialResponse.builder()
                                 .userId(TEST_USER_ID)
+                                .verifiableCredentials(List.of(SignedJWT.parse(SIGNED_VC_1)))
```

Comment:
I don't think you should need to add this - a `pending` response wouldn't have a VC attached to it.